### PR TITLE
Adaptability, Aerilate, Aftermath tests

### PR DESCRIPTION
**Pull request by kittenchilly**
## Description
Just a few tests, also found Aerilate working incorrectly

## **Discord contact info**
kittenchilly

### CODE REVIEW

File: test/battle/ability/aerilate.c [hunk 1]
```diff
@@ -21,7 +21,30 @@ SINGLE_BATTLE_TEST("Aerilate turns a Normal-type move into Flying-type move")
     }
 }
 
-TO_DO_BATTLE_TEST("Aerilate can not turn certain moves into Flying type moves");
+SINGLE_BATTLE_TEST("Aerilate can not turn certain moves into Flying type moves")
+{
+    u32 move;
+    PARAMETRIZE { move = MOVE_WEATHER_BALL; }
+    // PARAMETRIZE { move = MOVE_NATURAL_GIFT; } can't hold berry
```
Comment (AsparagusEduardo) [inline test/battle/ability/aerilate.c:28]: This could be solved using Skill Swap

**Review by AsparagusEduardo — APPROVED** [review]
Just one suggestion. Everything else lgtm :)

**Review by AsparagusEduardo — CHANGES_REQUESTED** [review]
Just one suggestion. Everything else lgtm :)

File: test/battle/ability/aerilate.c (hunk 1 above)
Comment (AsparagusEduardo) [inline test/battle/ability/aerilate.c:28]: Just one suggestion. Everything else lgtm :)

Comment (kittenchilly) [reply test/battle/ability/aerilate.c:28]: That's a pain, I'd have to make another test

Comment (AsparagusEduardo) [reply test/battle/ability/aerilate.c:28]: Alright, then let's just do this:
```suggestion
    // PARAMETRIZE { move = MOVE_NATURAL_GIFT; } TODO: handle this case via Skill Swap
```

**Comment by kittenchilly** [comment]
Should be done

**Review by AsparagusEduardo — APPROVED** [review]
Thank you! :D

**Pull request merged**